### PR TITLE
Implemented final touches

### PR DESCRIPTION
**Pull request by vangelif**
# In this PR, I:

- Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Made sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Made sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
- Improved the look and feel of the application.
- Kept the general layout of the wireframe.
- Used SCSS.
- Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### CODE REVIEW

File: dist/index.html
Comment (aamirkhan2478) [inline dist/index.html:1]: [Your code reference](https://github.com/vangelif/LB/blame/e3d9ffb010e6cca127141069008db17efba7ed45/src/index.js#LL11C1-L11C1)

- [x] I noticed your API key is not your generated API key so kindly generate your unique API key by using this [API tester](https://reqbin.com/) :+1:  check below :point_down: my video for more details :slightly_smiling_face: 

[screen-capture (6).webm](https://github.com/vangelif/LB/assets/45226848/2c110194-2595-400c-84ba-cc0c36ecf2bb)

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @vangelif , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html
Comment (vangelif) [reply dist/index.html:1]: Despite that the comment of this review is out of requirements' scope for today's project, you are kindly requested to submit your video format so I can refer to.

**Review by AbelG101 — APPROVED** [review]
# Hi @vangelif 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI is looking fabulous ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- [OPTIONAL] I'd suggest you name your branches to have a name of a feature or action (e.g. 'created_final_styling' or 'added_final_touches'). 😉

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**